code/headers/mlx90640_processor.hpp

```diff
@@ -53,10 +55,7 @@ namespace r2d2::thermal_camera {
          */
         void calculate_pixel_value(pixel_manipulator_c &manipulator);
 
-        static constexpr int KTA = 3;
-        static constexpr int KV = 2;
-        static constexpr int ALPHA = 1;
-        static constexpr int PIX_OS_REF = 0;
+        static constexpr int MLX_MAX_CLOCK_SPEED = 1'000'000;
```

Comment:
Is a unsigned int not better for this type of variable? This maximum speed shouldn't go below 0 i think